File: WCPhotoManipulator/MimeUtils.m
//
//  MimeUtils.m
//  WCPhotoManipulator
//
//  Created by Woraphot Chokratanasombat on 25/3/19.
//  Copyright © 2019 Woraphot Chokratanasombat. All rights reserved.
//

#import "MimeUtils.h"

@implementation MimeUtils

+ (NSString *)PNG {
    return @"image/png";
}

+ (NSString *)JPEG {
    return @"image/jpeg";
}

+ (NSString *)toExtension:(NSString *)type {
    if ([type isEqual:MimeUtils.PNG]) return @".png";
    else return @".jpg";
}

@end

File: WCPhotoManipulator/MimeUtils.swift
//
//  MimeUtils.swift
//  WCPhotoManipulator
//
//  Created by Woraphot Chokratanasombat on 28/7/20.
//  Copyright © 2020 Woraphot Chokratanasombat. All rights reserved.
//
import Foundation

@objc public class MimeUtils: NSObject {
    @objc public static let PNG = "image/png"
    @objc public static let JPEG = "image/jpeg"

    @objc public class func toExtension(_ type: String) -> String {
        if type == PNG {
            return ".png"
        }
        return ".jpg"
    }
}
